project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject agent "0.0.0"
```

Comment:
Revert this change because the version is computed on build

Reply:
Done